JunkyardLoad/JunkyardLoad.cs

```diff
@@ -39,8 +40,6 @@ public static async Task JunkyardNetcore31Metrics([TimerTrigger(MetricsInterval)
             await GetMetrics("dd-dotnet-linux-latest-build-stats", service: "net8-linux-latest-build-stats", log);
             await GetMetrics("dd-dotnet-linux-latest-build-profiler-default", service: "net8-linux-latest-build-profiler-default", log);
             await GetMetrics("dd-dotnet-linux-latest-build-profiler-all", service: "net8-linux-latest-build-profiler-all", log);
-            await GetMetrics("dd-dotnet-linux-latest-build-profiler-all", service: "net8-linux-latest-build-profiler-all", log);
-            await GetMetrics(SecurityAppUrlPrefix, SecurityService, log);
```

Comment:
You still want the ASM metrics don't you?

Reply:
I'll add them in another PR, as there's no endpoint yet in the sample